### DISCUSSION TITLE
All the unit test on Linux/macOS passed, but output "Expected" and "But Was"

### DISCUSSION THREAD

**Discussion by BDisp** [tui-cs/Terminal.Gui#1303] [Q&A] [answered]
@tznind do you know why there are these outputs?

![unit-tests](https://user-images.githubusercontent.com/13117724/118361047-89120400-b581-11eb-90b5-dcd9d45e289d.png)

It also happens on macOS.

![unit-tests-mac](https://user-images.githubusercontent.com/13117724/118371538-f4b99880-b5a4-11eb-9caa-b43c32ab6453.png)

**Comment by tznind** [comment, marked as answer]
Oops 

https://github.com/migueldeicaza/gui.cs/blob/a8628e7c28f7305d675e7660121a1540150644c9/UnitTests/GraphViewTests.cs#L111-L112

Looks like I need to implement `ITestOutputHelper` instead https://xunit.net/docs/capturing-output
